### PR TITLE
Update metadata-querying.md

### PR DESCRIPTION
**Pull request by jeremyyeo**
## Description & motivation
Single tenant deployments do not have access to the metadata API.

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [x] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

### CODE REVIEW

**Comment by mirnawong1** [comment]
hey @jeremyyeo thanks for opening this up and apologies for the delay here. since this is still an active clarification that applies and this pr has merge conflicts, do you mind opening up a new pr by editing this page `[here](https://docs.getdbt.com/docs/dbt-cloud-apis/metadata-querying)` to bypass it? i can open up the pr for you too, but i also would prefer you get the contribution credit too!

I've suggested the below tweaks from your original pr:

----
In order to query the dbt Metadata API, you’ll need to meet the following:

- Have a _Team_ or _Enterprise_ dbt Cloud  account
- Have a [multi-tenant](/docs/deploy/regions) dbt Cloud account
     * Note: The Metadata API is currently not available to accounts on [Single Tenant deployments](/docs/deploy/single-tenant).

File: website/docs/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-querying.md [hunk 1]
```diff
@@ -2,7 +2,9 @@
 title: "Querying the Metadata API"
 id: "metadata-querying"
 ---
-Accounts on the _Team_ and _Enterprise_ plans can query the dbt Metadata API.
+Accounts on the _Team_ and _Enterprise_ plans can query the dbt Metadata API. 
+
+_Note that this API is currently not available to accounts on [Single Tenant deployments](https://docs.getdbt.com/docs/dbt-cloud/deployments/single-tenant-deployment)._
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-querying.md]:
```suggestion
_Note that this API is currently not available to accounts on [Single Tenant deployments](/docs/deploy/single-tenant)._
```

**Review by runleonarun — DISMISSED** [review]
fix link the :shipit:

File: website/docs/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-querying.md [hunk 2]
```diff
@@ -2,7 +2,9 @@
 title: "Querying the Metadata API"
 id: "metadata-querying"
 ---
-Accounts on the _Team_ and _Enterprise_ plans can query the dbt Metadata API.
+Accounts on the _Team_ and _Enterprise_ plans can query the dbt Metadata API.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-querying.md]: Mirna's suggestions:
```suggestion
In order to query the dbt Metadata API, you must have the following:
* Team or Enterprise dbt Cloud account
* [Multi-tenant](https://github.com/docs/deploy/regions) dbt Cloud account
  Note: The Metadata API is currently not available to accounts on [Single Tenant deployments](/docs/deploy/single-tenant).
```

File: website/docs/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-querying.md [hunk 3]
```diff
@@ -2,7 +2,12 @@
 title: "Querying the Metadata API"
 id: "metadata-querying"
 ---
-Accounts on the _Team_ and _Enterprise_ plans can query the dbt Metadata API.
+In order to query the dbt Metadata API, you must have the following:
+* Team or Enterprise dbt Cloud account
+* [Multi-tenant](https://github.com/docs/deploy/regions) dbt Cloud account
+  Note: The Metadata API is currently not available to accounts on [Single Tenant deployments](/docs/deploy/single-tenant).
+
+_Note that this API is currently not available to accounts on [Single Tenant deployments](/docs/deploy/single-tenant)._
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-querying.md]:
```suggestion
```

File: website/docs/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-querying.md [hunk 4]
```diff
@@ -2,7 +2,11 @@
 title: "Querying the Metadata API"
 id: "metadata-querying"
 ---
-Accounts on the _Team_ and _Enterprise_ plans can query the dbt Metadata API.
+In order to query the dbt Metadata API, you must have the following:
+* Team or Enterprise dbt Cloud account
+* [Multi-tenant](https://github.com/docs/deploy/regions) dbt Cloud account
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-querying.md]:
```suggestion
* [Multi-tenant](/docs/deploy/regions) dbt Cloud account
```

**Review by mirnawong1 — APPROVED** [review]
updated snippet to callout ST, which is going to be supported soon per cloud feature parity table

**Pull request merged**